### PR TITLE
Wf categories fieldgroup

### PR DESCRIPTION
**Pull request by rammatta**
Please link to the issue #…
Adding a creating a new field group and referencing that field group in another field group.

### CODE REVIEW

File: extensions/adobe/experience/workfront/categoryparameter.schema.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/experience/workfront/object",
+    "https://ns.adobe.com/experience/workfront/changeevent"
+  ],
+  "description": "Category Parameter related fields",
+  "definitions": {
+    "wfobject": {
```
Comment (mjasrotia) [inline extensions/adobe/experience/workfront/categoryparameter.schema.json:20]: Not an issue but I would suggest to change the definition name here to something like "category-parameter" . I guess it's a copy paste from the previous field group.

File: extensions/adobe/experience/workfront/category.schema.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  "definitions": {
+    "wfobject": {
+      "properties": {
+        "workfront:category": {
+          "type": "object",
+          "title": "Category Attributes",
+          "properties": {
+            "workfront:description": {
+              "title": "Category Description",
+              "description": "",
```
Comment (mjasrotia) [inline extensions/adobe/experience/workfront/category.schema.json:28]: None of the properties have a description. Please provide a context if possible

File: extensions/adobe/experience/workfront/categoryparameter.schema.json (hunk 1 above)
Comment (rammatta) [reply extensions/adobe/experience/workfront/categoryparameter.schema.json]: I made this change, can you provide a little more detail of where this field/attribute is being used?

File: extensions/adobe/experience/workfront/category.schema.json (hunk 2 above)
Comment (rammatta) [reply extensions/adobe/experience/workfront/category.schema.json]: added description to the columns

File: extensions/adobe/experience/workfront/category.schema.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+              "type": "string"
+            }
+          }
+        }
+      }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "#/definitions/wfobject"
```
Comment (mjasrotia) [inline extensions/adobe/experience/workfront/category.schema.json:53]: This needs to match the local definition

File: extensions/adobe/experience/workfront/categoryparameter.schema.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+              "type": "string"
+            }
+          }
+        }
+      }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "#/definitions/wfobject"
```
Comment (mjasrotia) [inline extensions/adobe/experience/workfront/categoryparameter.schema.json:58]: This needs to match the local definition

Comment (rammatta) [reply extensions/adobe/experience/workfront/categoryparameter.schema.json]: this is updated

File: extensions/adobe/experience/workfront/category.schema.json (hunk 3 above)
Comment (rammatta) [reply extensions/adobe/experience/workfront/category.schema.json]: This is updated

**Review by mjasrotia — APPROVED** [review]

**Comment by cmathis** [comment]
retest

**Review by mjasrotia — APPROVED** [review]

**Pull request merged**